browser/views.py

```diff
@@ -687,7 +707,6 @@ def delete_investigator_record(request, person_id, research_id):
     return HttpResponseRedirect(reverse('person', args=(person_id,)))
 
 
-@staff_member_required
 def position(request, person_id, position_id=None):
```

Comment:
why is this annotation being deleted?

Reply:
This was deleted accidentally. I added the annotation back.